spec/lib/gakubuchi/engine_registrar_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe Gakubuchi::EngineRegistrar do
+  let(:env) { Sprockets::Environment.new }
+  let(:engine_registrar) { described_class.new(env) }
+
+  describe '#register!' do
```

Comment:
[IMO]
Reconsider test cases for this method referring `spec/lib/gakubuchi/template_engine_spec.rb`.